fix/poes_sst.navy_avhrr

```diff
@@ -15,9 +15,8 @@ echo " Starting $TASK and  $TASK.$family "
       eval REMOTEDSNGRP${n}=\${REMOTEDSNGRP${n}:-'"concatenate_families \
 mcsst_mtc_d*tmp \
 mcsst_mtb_d*tmp \
-mcsst_mta_d*tmp"'}
-#mcsst_npp_d*tmp \
-#mcsst_n20_d*tmp \
+mcsst_npp_d*tmp \
```

Comment:
When you tested this, did you save the b012/xx012 tanks with extra data? Path?

Reply:
I am testing this now.  Will let you know when it is done and point the dir to you.  Thanks.

Reply:
Hello,  Here is where the tank is for the test ingest:
[clogin02 /lfs/h2/emc/obsproc/noscrub/steve.stegall/DCOMDIR/RB_4.5_test/prod/20230104/b012]$ ll
total 5001076
-rw-r--r-- 1 steve.stegall obsproc 2559120464 Jan  4 17:05 xx012
-rw-r--r-- 1 steve.stegall obsproc 2486921832 Jan  4 17:32 xx023
-rw-r--r-- 1 steve.stegall obsproc   74967016 Jan  4 17:06 xx222

Reply:
Thanks! I'll wait for the user to chime in on their needs...